Source/objects.cpp

```diff
@@ -2632,6 +2632,7 @@ void OperateL2RDoor(int pnum, int oi, BOOL sendflag)
 	}
 }
 
+// BUGFIX (?) - out of all door functions, only this one is BOOL in hellfire, apparently for no reason

```

Comment:
Don't change anything logically so no need for the comment I think. Just inconsistent code style and there is plenty of that :D

Reply:
But it's good to have it noted here in the PR :)

Reply:
yeah it just looks odd and affects the asm so ... : p